arch/arm/boot/dts/bcm2708-rpi-zero-w.dts

```diff
@@ -3,6 +3,7 @@
 #include "bcm2708.dtsi"
 #include "bcm2708-rpi.dtsi"
 #include "bcm283x-rpi-csi1-2lane.dtsi"
+#include "bcm283x-rpi-i2c0mux_0_28.dtsi"
```

Comment:
So I don't leap to the wrong conclusion, is there a reason why bcm2708-rpi-zero.dts doesn't get the same treatment? Rev 1.2 hardware didn't have the connector, but rev 1.3 does.

Reply:
I suspect I just missed it.
Having the mux does no harm on the 1.2, and it has benefit on the 1.3, so better to have it in there.